starter-guides/nodejs/index.js

```diff
@@ -0,0 +1,18 @@
+const { LitNodeClient } = require('@lit-protocol/lit-node-client');
+
+const connectToLit = async () => {
+    try {
+      // More information about the available Lit Networks: https://developer.litprotocol.com/category/networks
+      const litNodeClient = new LitNodeClient({
+        litNetwork: 'datil-dev',
```

Comment:
We should start being consistent and only use the constants package instead of specifying the string value manually

Reply:
Sounds good, this has been changed